wire-ios/Wire-iOS Share Extension/Resources/Localization/ar.lproj/Localizable.strings

```diff
@@ -42,24 +45,35 @@
 "share_extension.unlock.alert.message" = "Open Wire to create a passcode";
 "share_extension.unlock.reveal_passcode" = "Reveal passcode";
 
+// Timeout
 "share_extension.timeout.title" = "انتهت مهلة الاتصال.";
 "share_extension.timeout.message" = "تحقق من اتّصالك بالإنترنت ثم حاول مجددًا. المرفقات حفظت في المحادثة.";
 
-"content.file.too_big" = "يمكنك إرسال ملفات حتى %@";
-"content.file.unsupported_attachment" = "Unsupported attachment";
+// Content
+"share_extension.content.file.too_big" = "يمكنك إرسال ملفات حتى %@";
+"share_extension.content.file.unsupported_attachment" = "Unsupported attachment";
```

Comment:
Looks like the Share Extension strings haven't gotten updates recently. Seems to be fixed now due to this change:
https://github.com/wireapp/wire-ios/pull/2603/files#diff-256aa18d225b85dd7da40aba5091225c2b6c93e8e40f8955fc92be6c5a674afcL37-R39